plugin.js

```diff
@@ -25,6 +25,19 @@ function fastifyCookieSetCookie (reply, name, value, options, signer) {
   }
 
   const serialized = cookie.serialize(name, value, opts)
+
+  if (opts.enableWarnOnSafeLimit === true) {
+    const limit = opts.warnOnSafeLimit || 4096
```

Comment:
Make sure `opts.warnOnSafeLimit` is always set.

Reply:
I guess this is related to the proposed change on line 29?

Reply:
Yes, but it will likely need more work!